src/RSEF/object_creator/implementation_url.py

```diff
@@ -42,14 +45,14 @@ def extraction_methods(self):
 
     @extraction_methods.setter
     def extraction_methods(self, value):
-        self._extraction_methods = value
+        self._extraction_methods = [ExtractionMethod.from_dict(em) for em in value]
 
     def to_dict(self):
         return {
-            "identifier": self.identifier,
-            "type": self.type,
-            "paper_frequency": self.paper_frequency,
-            "extraction_methods": [em for em in self.extraction_methods] 
+            "identifier": self._identifier,
+            "type": self._type,
+            "paper_frequency": self._paper_frequency,
+            'extraction_methods': [em.to_dict() for em in self._extraction_methods] 
```

Comment:
Please make this double quotes to folllow the same coding style

Reply:
Fixed in the new commit